string tagName
uint8 valueToWrite
---
uint8 ret
